app/Resources/patterns/hidden-input.mustache

```diff
@@ -1 +0,0 @@
-<input type="hidden" {{#id}}id="{{id}}"{{/id}} {{#name}}name="{{name}}"{{/name}} {{#value}}value="{{value}}"{{/value}}>
```

Comment:
Appear to have not updated for https://github.com/elifesciences/patterns-php/pull/287.